template/oss-npm-package/_package.json

```diff
@@ -38,6 +38,7 @@
     "commitizen": "^4.2.4",
     "skuba": "*"
   },
+  "packageManager": "pnpm@8.9.0",
```

Comment:
Worth discussing if we want to pin a particular pnpm version, and if so, whether we should express it in this way.

This is supported by `pnpm/action-setup` in GitHub Actions, but I don't think it'll work with [Corepack](https://pnpm.io/installation#using-corepack) in our Dockerfiles elsewhere.